orbs/phraseapp/README.md

```diff
@@ -0,0 +1,65 @@
+# Phraseapp Orb
+
+## Proposal
+This orb was created to make the merge of the changes made in the app.phrase with the GitHub project repository simple.
+
+## Configuration Needed
+1) Add [pharaseapp.yml](.phraseapp-sample.yml) to the root of your project.
+It will be used to know what locales we need to fetch from the app.phrase, the path to pull the changes and the format of the configuration file.
+```
+phraseapp:
+  available_locales:
+    en
+    fr
+    de
+  pull:
+    targets:
+      - file: ./config/locales/
+        params:
+          file_format: yml
+```
```

Comment:
Might be worth linking to examples of config in different projects with different languages. The format will typically be different in Go, Ruby, JS...